src/cache-utils.js

```diff
@@ -69,7 +69,9 @@ function blockTagParamIndex (payload) {
   }
 }
 
+// eslint-disable-next-line consistent-return
 function cacheTypeForPayload (payload) {
+  // eslint-disable-next-line default-case
```

Comment:
We should avoid disabling rules when we can, and I believe we resolve both rule violations without changing the function's behavior by adding the following default case:
```javascript
default:
  return undefined
```

Reply:
We can then default to `'never'` in a follow-up, as discussed.

Reply:
Issue for the follow-up: https://github.com/MetaMask/eth-json-rpc-middleware/issues/65